test/test_distributions.py

```diff
@@ -791,6 +791,24 @@ def test_has_examples(self):
                 self.assertIn(Dist, distributions_with_examples,
                               "Please add {} to the EXAMPLES list in test_distributions.py".format(Dist.__name__))
 
+    def test_distribution_expand(self):
+        shapes = [torch.Size(), torch.Size((2,)), torch.Size((2, 1))]
+        for Dist, params in EXAMPLES:
+            if Dist.__name__ == "TransformedDistribution":
```

Comment:
Cannot this be `isinstance(Dist, TransformedDistribution)` ?

Reply:
We subclass `TransformedDistribution` in many cases, and I believe Neeraj is ensuring that those subclasses are tested.